index.ts

```diff
@@ -124,27 +138,26 @@ module.exports = {};`
 
     this.name = pkg.name;
     this.version = pkg.version;
-    this.keywords = pkg.keywords;
```

Comment:
I dropped this because it wasn't tested and looks like it wasn't fully implemented.